### ISSUE TITLE
Avatar Images

### ISSUE THREAD

**Issue by yiiTT** [ewhal/nyaa#379]
- Allow users to use image URLs from imgur.com and mixtape.moe (any more?)
- Strict rules on max avatar resolution and file size anyone breaking the rules will be banned on sight.

When signing up or changing account settings there should be a field to enter a image URL. If the image URL field is empty then use Gravatar.  Also there should be a check box to set avatar to default (use ID 0 - れんちょん's avatar) if they don't want to use Gravatar.

**Comment by thewisenerd** [comment]
instead of having strict rules on max avatar resolution and filesize, can't we let the server deal with it? iirc, imgur already supports auto resized images in small square, big square, etc; thus, we could scale all uploaded images to a fixed "square" resolution from the image host side.

_p.s. assuming the user doesn't use a square image, with a bit of CSS, we can just get that centered; because, hey; the user decided not to use a square image... (or deal with it in the image host, either way.)_

**Comment by ayame-git** [comment]
>let the server deal with it
Can't we just set size in frontend?

**Comment by ayame-git** [comment]
Nice fucking formatting github

**Comment by thewisenerd** [comment]
nah, having it set in the front end will definitely increase user load, if some user decides (against all shred of reason) that having a 1MB profile picture is the way to go.

**Comment by jithatsonei** [comment]
can we blacklist mixtape.moe instead and just force doko.moe

**Comment by thewisenerd** [comment]
is the reason for this because images uploaded to mixtape.moe get "lost" ?
